onnx_chainer/functions/normalization.py

```diff
@@ -98,3 +98,17 @@ def convert_LocalResponseNormalization(func, onnx_op_name, opset_version,
             bias=float(func.k),
             size=size,
         ),
+
+
+def convert_NormalizeL2(func, onnx_op_name, opset_version, input_names,
+                        output_names, parameters):
+    if isinstance(func.axis, tuple) and len(func.axis) != 1:
+        raise ValueError(
+            'multiple axes ({}) is not supported in ONNX\'s LpNormalization '
+            'operation'.format(func.axis))
+    if opset_version == 1:
```

Comment:
How about checking if `eps` is 1e-5? ONNX does not have `eps` so we cannot reproduce Chainer's result by ONNX's LpNormalization, but I think it's probably OK. The default `eps` is just an implementation detail of a framework which makes models a bit more robust. However, if a user specifies a non-default value explicitly, I think it's better to raise an exception to notify the user onnx-chainer cannot reproduce the user's intention.

Reply:
thx, add checker on `eps` but I don't know using `sys.float_info.espsilon` is Chainer way or not.